_redirects

```diff
@@ -1 +1,2 @@
 https://fall.sparkdesignsystem.com/* https://staging.sparkdesignsystem.com/:splat 301!
+https://sparkdesignsystem.com/gettingstarted/developers https://www.sparkdesignsystem.com/ 301!
```

Comment:
should we do the same for the designers page that we deleted?

Reply:
Ope! 😂